examples/cmd/snapshotting/go.mod

```diff
@@ -36,7 +36,8 @@ require (
 	github.com/vishvananda/netlink v1.2.1-beta.2 // indirect
 	github.com/vishvananda/netns v0.0.0-20211101163701-50045581ed74 // indirect
 	go.mongodb.org/mongo-driver v1.10.0 // indirect
-	golang.org/x/net v0.0.0-20220728211354-c7608f3a8462 // indirect
+	golang.org/x/net v0.10.0 // indirect
```

Comment:
The PR mentions golang.org/x/net latest version as `v0.12.0` but this is still `v0.10.0`?

Reply:
I was planning to upgrade to latest which is v0.12.0, then told by CI that some dependencies introduced by v0.12.0 cannot be built with current go version. 
So downgraded to v0.10.0.  
Yet noticed another issue which requires go 1.17. 
I will upgrade that first.

Reply:
I have PR #498 upgrading go to 1.17 merged.
And switched back to upgrade target version to v.012.0 which is latest.